metricflow/model/semantics/linkable_spec_resolver.py

```diff
@@ -561,7 +679,7 @@ def _get_linkable_element_set_for_measure(self, measure_reference: MeasureRefere
                     )
                 )
         all_linkable_elements = LinkableElementSet.merge_by_path_key(
-            [local_linkable_elements]
+            [metric_time_linkable_element_set, local_linkable_elements]
```

Comment:
Added bonus, if we factor the logic into a method where this is defined becomes much more obvious.